catppuccinpalette.dtx

```diff
@@ -0,0 +1,609 @@
+% \iffalse meta-comment
+%
+% Copyright (C) 2024 by Lukas Heindl <oss.heindl+latex@protonmail.com>
+% ---------------------------------------------------------------------------
+% This work may be distributed and/or modified under the
+% conditions of the LaTeX Project Public License, either version 1.3
+% of this license or (at your option) any later version.
+% The latest version of this license is in
+%   http://www.latex-project.org/lppl.txt
+% and version 1.3 or later is part of all distributions of LaTeX
+% version 2005/12/01 or later.
+%
+% This work has the LPPL maintenance status `maintained'.
+%
+% The Current Maintainer of this work is Lukas Heindl.
+%
+% This work consists of the files catppuccinpalette.dtx and catppuccinpalette.ins
+% and the derived filebase catppuccinpalette.sty.
```

Comment:
The license that we prefer across the organisation is MIT, would you be willing to re-license this under the organisation as specified in our [LICENSE](https://github.com/catppuccin/latex/blob/main/LICENSE)

Reply:
Oh yes I've forgotten this in my todos above, licensing and the author specified in the docs are still to be discussed.